yarn-project/acir-simulator/src/index.test.ts

```diff
@@ -26,7 +26,7 @@ type NoirPoint = {
   y: bigint;
 };
 
-export const createMemDown = () => (memdown as any)();
+export const createMemDown = () => (memdown as any)() as MemDown<any, any>;
```

Comment:
Could have 
```
export const createMemDown = (): MemDown<any, any>  => (memdown as any)();
```

Reply:
True, but coming from an any it doesn't make a difference